storage/persistence.go

```diff
@@ -18,6 +20,8 @@ func (b *Bucket) UploadBuildMetadata(ctx context.Context, build model.Build) err
 	return errors.Wrapf(b.Put(ctx, metadata.key(), bytes.NewReader(json)), "putting metadata for build '%s'", build.Id)
 }
 
+// UploadBuildMetadata uploads metadata for a new test to the offline blob
```

Comment:
what do we mean by "offline" blob bucket storage?

Reply:
Basically that this is not on the app or the application DB.